helm-chart/renku/templates/gateway/gitlab-redirect-ingress.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.gateway.redirects.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: gitlab-redirect-{{ template "renku.fullname" . }}
+  labels:
+    app: gitlab
+    chart: {{ template "renku.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+    nginx.ingress.kubernetes.io/proxy-body-size: "0"
+    nginx.ingress.kubernetes.io/proxy-buffer-size: 8k
+    nginx.ingress.kubernetes.io/proxy-request-buffering: "off"
+    nginx.ingress.kubernetes.io/use-regex: "true"
+    nginx.ingress.kubernetes.io/rewrite-target: /api/gitlab-redirect/$1
+spec:
+  ingressClassName: webapprouting.kubernetes.azure.com
```

Comment:
issue: This needs to be modifiable through the values file. Because if someone is running on another cloud or infra they will need to change it to what their provider has.

I would put an `ignress` field under `redirects` (or `gitlabRedirects`) in the values file  and add this and a few other configs with defaults for what we use.

Reply:
I am referring to the `ingressClassName` field here specifically.